newIDE/app/src/MainFrame/Preferences/PreferencesProvider.js

```diff
@@ -281,6 +283,37 @@ export default class PreferencesProvider extends React.Component<Props, State> {
     return preferences;
   }
 
+  _getLastUsedPath(project: gdProject, kind: ResourceKind) {
+    const projectName = project.getName();
+    const values = this.state.values;
+    const curProjectPaths = values.lastUsedPath[projectName];
```

Comment:
nitpicking: curProjectPaths => projectPaths

"cur" does not give much extra information.